smart_open/tests/test_s3.py

```diff
@@ -460,6 +460,30 @@ def test_list_bucket_long(self):
         expected = ['key_%d' % x for x in range(num_keys)]
         self.assertEqual(sorted(keys), sorted(expected))
 
+    @mock.patch('boto3.client')
+    @mock.patch('boto3.session.Session')
+    def test_credentials(self, mock_session, mock_client):
+        # setup mocks
+        mock_s3 = mock.Mock()
+        mock_client.return_value = mock_s3
+        mock_s3.list_objects_v2.return_value = {'Contents': [{'Key': 'test_key'}]}
+
+        # make call
+        # Note we disable multiprocessing since mocks only work in the thread.
+        list(smart_open.s3.iter_bucket(BUCKET_NAME,
```

Comment:
Please use a hanging indent, here and everywhere.

Reply:
I was just following the style I saw used in `iter_bucket`'s function definition. Refactored the function calls to use hanging indent. Would you like the function definitions refactored as well?

Reply:
Yes, please make sure any _new_ code using hanging indents.

Don't worry about reformatting code you have not changed.